gallery.html

```diff
@@ -256,6 +298,49 @@
 <!--[if lte IE 8]>
 <script src="assets/js/ie/respond.min.js"></script><![endif]-->
 <script src="assets/js/main.js"></script>
+<script>
+    $('document').ready(function() {
+        var items = $('.item');
+        var pageCount = Math.ceil(items.length/5);
+        var first = template('|<<', -1);
+        var last = template('>>|',0);
+
+        $(items).hide();
+        for(var i = 0; i < 5; i++){
+            $(items[i]).show();
+        }
+
+        $('#pagination').find('ul').append(first);
```

Comment:
yeah, i just don't want to study... and the libs has their own styles. don't want to bother to change the style things